index.html

```diff
@@ -2,36 +2,37 @@
 <html lang="en">
 
 <head>
-  <title>Document</title>
-  <link rel="stylesheet"
-    href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
-    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
-    crossorigin="anonymous" />
+  <title>OnSwipe Careers</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous" />
+  <!--Main stylesheet-->
+  <link rel="stylesheet" href="style.css" />
   <!-- Mapbox stylesheet -->
-  <link href='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.css'
-    rel='stylesheet' />
+  <link href='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.css' rel='stylesheet' />
+
 
-  <link rel="stylesheet" href="style.css" />
 </head>
 
 <body>
-  <section>
-    <div class="container">
-        <div class="row">
-          <div class="col-12 header text-center">
-            <h2 class="display-2">onSwipe</h2>
-            <p>Tinder for Tech Jobs</p>
-            <a href="favorites.html">Favorite List</a>
-          </div>
-        </div>
-        <div class="row">
-          <div class="col-12 results">
-            <!-- Job-Cards Show Up Here! -->
-            <div class="jobcards-container"></div>
+  <div class="wrapper">
 
-          </div>
+  <header>
+    <div class="container1">
```

Comment:
"container" is a bootstrap class so it needs to stay on this div. Can we add it back and then we can also keep "container1." Let me know, thanks!

Reply:
I don't think we need it. I had to override some of the bootstrap classes in order to style them in the CSS. I believe that I replaced that section with other elements. I changed the class so it didn't conflict with any other container classes that may have existed. I  tried to avoid using bootstrap as much as possible, because it limits the styling.

Reply:
Gotcha. I thought it needed to be container for the bootstrap grids and rows, but if everything else is still working as intended then that works for me!